addons/power/functions/fnc_turnOffSolarAction.sqf

```diff
@@ -0,0 +1,35 @@
+/*
+ * Turns off the solar panel.
+ *
+ * Arguments:
+ * 0: Solar panel <OBJECT>
+ * 1: If the ace progress bar is shown <BOOL>
+ * 
+ * Return:
+ * None
+*/
+params ["_entity", ["_silent", false]];
+
+if (!_silent) then 
+{
+	[
+		3,
+		[_entity], 
+		{
+
+		},
+		{},
+		("Turn Off")
+	] call ace_common_fnc_progressBar;
+};
+
+[_entity, true, [0, 1, 0], 0] call ace_dragging_fnc_setDraggable;
+
```

Comment:
//We should add something like this, so the checkPowerGenAction will return 0 Watts if the solar panel is powered down.

_entity setVariable ['AE3_power_powerCapacity', 0, true];